assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -77,6 +82,35 @@ const Duration = styled.div`
   padding: 2px 8px;
   border-radius: 8px;
 `;
+
+const gradientAnimation = keyframes`
+    0% { background-position:0% 50% }
+    50% { background-position:100% 50% }
+    100% { background-position:0% 50% }
+`;
+
+const UploadingIndicator = styled.div`
+  height: 4px;
+  background: linear-gradient(270deg, #4285f4 15%, #15d8fd 50%, #4285f4 85%);
```

Comment:
Should these values be in the theme?

Reply:
Agreed! Should I replace them in other parts that we are using them statically (e.g. `#4285f4`) in this PR?